package.json

```diff
@@ -62,7 +62,7 @@
     "typescript": "^4.9.5"
   },
   "dependencies": {
-    "@jellyfish-dev/membrane-webrtc-js": "^0.6.3",
+    "@jellyfish-dev/membrane-webrtc-js": "github:jellyfish-dev/membrane-webrtc-js#master",
```

Comment:
Just to make sure, we do not want to release a new version of `membrane-webrtc-js` beforehand?